assets/blocks/quiz/data.js

```diff
@@ -172,3 +202,18 @@ export const normalizeAttributes = ( options, mapFunction ) => {
 		return value;
 	} );
 };
+
+/**
+ * Checks whether a bock is empty.
```

Comment:
Typo for `block`

Reply:
🐔 🐔 🐔

Reply:
2e6bc76be36de7e518ea27e877ca8c8534d2a424